e2e/test/scenarios/actions/model-actions.cy.spec.js

```diff
@@ -591,10 +603,16 @@ describe(
         cy.findByText("Actions").click();
       });
 
-      cy.findByRole("button", { name: /Create basic actions/i }).click();
+      createBasicActions();
 
       openActionEditorFor("Create");
 
+      cy.wait("@getAction").then(({ response }) => {
+        const { parameters, visualization_settings } = response.body;
+        expect(parameters).to.have.length(5);
+        expect(visualization_settings).to.have.property("fields");
+      });
```

Comment:
This test was failing because the `GET` action api request was return an action without parameters or visualization_settings, so there was no form to render at all. I'm not sure this wait actually helps, but I couldn't reproduce the failure again after I added it. In any event, these assertions themselves should fail and make the cause of the failure clearer than just "form didn't show up"

fail | pass
--- | ---
![Screen Shot 2023-07-26 at 3 06 41 PM](https://github.com/metabase/metabase/assets/30528226/8a71da5a-38bd-4c44-95f6-fca9463b83f1) | ![Screen Shot 2023-07-26 at 3 07 27 PM](https://github.com/metabase/metabase/assets/30528226/a7c960f2-021b-4e70-ad7b-f4f08c5a906a)